backend/api/urls.py

```diff
@@ -2,8 +2,11 @@
 from . import views
 
 urlpatterns = [
-    path('add/', views.addEvent),
+    path('add/event/', views.addEvent),
     path('delete/<int:id>/', views.deleteEvent),
     path('get/<int:id>/', views.getEventByID),
     path('', views.getEvents),
+    path('get/non-registered-users/<int:id>/', views.get_non_registered_users_by_event_id),
+    path('add/non-registered-user/', views.add_non_registered_user),
+    path('get/non-registered-users/', views.get_non_registered_users)
```

Comment:
I want to clean up our endpoints later on, but for this PR it's fine to leave it as is.